crates/bevy_asset/src/io/memory.rs

```diff
@@ -55,6 +55,16 @@ impl Dir {
         );
     }
 
+    pub fn remove_asset(&self, path: &Path) -> Option<Data> {
```

Comment:
Doc strings here too :)

Reply:
Wanted to keep it consistent 😂

Reply:
Yeah but our current lack of docs in the siblings is *bad*!

Reply:
I was at least 50% kidding ;) will add them